tests/console/machinery.pm

```diff
@@ -0,0 +1,29 @@
+# SUSE's openQA tests
+#
+# Copyright © 2012-2016 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+use base "consoletest";
+use testapi;
+
+sub run() {
+    select_console 'root-console';
+
+    # add machinery repository from ymp file
+    assert_script_run "yast OneClickInstallCLI prepareinstall url=http://machinery-project.org/machinery.ymp targetfile=/tmp/local.ymp", 200;
+    assert_script_run "yast OneClickInstallCLI doinstall instructionsfile=/tmp/local.ymp", 200;
+
+    if (match_has_tag('import-untrusted-gpg-key')) {
```

Comment:
that call only makes sense after an assert_screen or check_screen. We should reset the last match for every test